lib/shopify_cli/cli.rb

```diff
@@ -31,6 +31,8 @@ def add(connection)
         puts "\nopen https://#{config['domain']}/admin/apps/private in your browser to create a private app and get API credentials\n"
         config['api_key']  = ask("API key?")
         config['password'] = ask("Password?")
+        config['version']  = ask("API Version? (leave blank for '2019-04')")
```

Comment:
is the long-term plan to hardcode this or this just temporary?

Reply:
We _could_ round to the nearest + previous quarter?